src/main/java/janggi/movement/crash/CrashOrBridgeAndPrey.java

```diff
@@ -0,0 +1,57 @@
+package janggi.movement.crash;
+
+import janggi.game.Team;
+import janggi.movement.route.Hurdles;
+import janggi.piece.Movable;
+import janggi.point.Point;
+import java.util.List;
+
+public final class CrashOrBridgeAndPrey extends Crashes {
+    private static final int ONLY_BRIDGE_COUNT = 1;
+    private static final int ONLY_BRIDGE_AND_PREY_COUNT = 2;
+
+    public CrashOrBridgeAndPrey(List<Point> crashes) {
+        super(crashes);
+    }
+
+    @Override
+    public boolean hasOnlyPassables(Team movingTeam, Point targetPoint, Hurdles hurdles) {
+        return isBridgeOnly(targetPoint, hurdles)
+                || isBridgeAndPreyOnly(movingTeam, targetPoint, hurdles);
+    }
+
+    private boolean isBridgeOnly(Point targetPoint, Hurdles hurdles) {
+        if (crashes.size() == ONLY_BRIDGE_COUNT) {
+            return isBridgeExists(targetPoint, hurdles);
+        }
+        return false;
+    }
+
+    private boolean isBridgeAndPreyOnly(Team movingTeam, Point targetPoint, Hurdles hurdles) {
+        if (crashes.size() == ONLY_BRIDGE_AND_PREY_COUNT) {
+            return (isBridgeExists(targetPoint, hurdles)
+                    && isPreyExists(movingTeam, targetPoint, hurdles));
+        }
+        return false;
+    }
```

Comment:
```suggestion
    private boolean isBridgeExists(Point targetPoint, Hurdles hurdles) {
        Point bridgePoint = crashes.getFirst();
        return !(hurdles.findByPoint(bridgePoint).isPo() || bridgePoint.equals(targetPoint));
    }
```

`if` 문을 사용하지 않고 그대로 리턴해도 동일합니다.

Reply:
엇 이해가 안가서 다시 여쭤보는데요, 31번째의 if문이 없어도 된다는 뜻이신가요?
if문이 없어진다면 isPreyExists에서 공격대상이 1번째 인덱스에 있어야 한다는 걸 확인하지 않기 때문에 브릿지와 공격대상 외에도 다른 장애물이 들어올 가능성이 허용되지 않나 싶었습니다..!